tests/integration-tests/conftest.py

```diff
@@ -684,6 +685,21 @@ def inject_additional_config_settings(cluster_config, request, region, benchmark
 
     if request.config.getoption("custom_ami") and not dict_has_nested_key(config_content, ("Image", "CustomAmi")):
         dict_add_nested_key(config_content, request.config.getoption("custom_ami"), ("Image", "CustomAmi"))
+    # Adding Internal AMI we create for integration testing of those OSses which we do not publicly publish/release
+    if config_content["Image"]["Os"] in PRIVATE_OSSES and not dict_has_nested_key(
+        config_content, ("Image", "CustomAmi")
+    ):
+        dict_add_nested_key(
```

Comment:
Nice approach. Does it permit to run integ tests for released version as well?

Reply:
> Does it permit to run integ tests for released version as well?

The function [retrieve_lates_ami](https://github.com/aws/aws-parallelcluster/blob/49131fbf848b1beedcbc2110bf233e01c2ebc6f1/tests/integration-tests/tests/common/utils.py#L106-108) will run against a released version of AMI if we dont use any Git refs

Reply:
I am testing the same by removing the `allow_private_ami` that i had kept as True for testing

Reply:
Tests successfully